env0/data_project.go

```diff
@@ -19,12 +19,14 @@ func dataProject() *schema.Resource {
 				Description:  "the name of the project",
 				Optional:     true,
 				ExactlyOneOf: []string{"name", "id"},
+				Computed:     true,
```

Comment:
Why would it fix the issue?
Strange...
output does work the second time running apply.

Reply:
I'm not sure but I wouldn't hold any thought on this one. Let's ship this out ASAP.
The provider framework is FAR from being great 😞